apps/subscription/models.py

```diff
@@ -0,0 +1,75 @@
+from django.db import models
+from apps.user.models import User
+
+# Create your models here.
+
+
+# Subject Data Model
+class Subject(models.Model):
+    name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.name
+
+
+# SubjectUsers is a bridge between subject and user
+class SubjectUsers(models.Model):
+    project = models.ForeignKey(
+        Subject,
+        db_index=True,
+        on_delete=models.CASCADE,
+        null=False,
+        blank=False
+    )
+
+    user = models.ForeignKey(
+        User,
+        db_index=True,
+        on_delete=models.CASCADE,
+        null=False,
+        blank=False
+    )
+
+    def __str__(self):
+        return ''
+
+    # fixes a pluralization issue and forces project and user to be unique together
+    class Meta:
+        unique_together = (
+            ('project', 'user'),
+        )
+        verbose_name_plural = 'Project Users'
+
+
+class Subscription(models.Model):
+
+    # Constants for sub status
+    SUBSCRIBED = 'subscribe'
```

Comment:
Nice use of constants. Allows to expand the subscription status beyond true or false. I don't see these used anywhere. Maybe here for future use?

What do you think about adding a date created and date updated field to your models? I always like to add them for audit purposes.